openassessment/xblock/staff_area_mixin.py

```diff
@@ -272,6 +272,32 @@ def render_staff_grade_form(self, data, suffix=''):  # pylint: disable=W0613
         except PeerAssessmentInternalError:
             return self.render_error(self._(u"Error getting staff grade information."))
 
+    @XBlock.handler
+    @require_course_staff("STUDENT_GRADE")
+    def render_staff_grade_counts(self, data, suffix=''):  # pylint: disable=W0613
```

Comment:
What is `suffix` used for here? It seems to be a common pattern so I doubt this needs to be fixed, I'm just curious.